### PR TITLE
Run say_when as a cloudwatch event cron

### PR DESCRIPTION
**Pull request by cavis**

### CODE REVIEW

File: stacks/apps/exchange.yml [hunk 1]
```diff
@@ -1147,7 +953,30 @@ Resources:
...
+    Condition: EnableWorkers # See README
+    Properties:
+      Description: !Sub Periodically runs ${EnvironmentType} SayWhen jobs
+      ScheduleExpression: rate(1 hour)
+      State: ENABLED
+      Targets:
+        - Arn: !Ref EcsClusterArn
+          EcsParameters:
+            TaskCount: 1
+            TaskDefinitionArn: !Ref WorkerTaskDefinition
```
Comment (cavis) [inline stacks/apps/exchange.yml:967]: Oh, I see.  Reuse the worker task def, but run a different command.

File: stacks/apps/exchange.yml [hunk 2]
```diff
@@ -1391,6 +1220,7 @@ Resources:
         - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
         - { Key: prx:dev:application, Value: Exchange }
       VisibilityTimeout: 5400 # 1.5 hour
+  # TODO: remove
```
Comment (cavis) [inline stacks/apps/exchange.yml:1223]: I guess this TODO is no longer valid either - I think I assumed we wouldn't need the queue at all anymore.  But if i understand this, the `process_jobs` will _add_ jobs to the say_when queue, to be processed by the other/real workers.  Right?

**Review by cavis — COMMENTED** [review]
I think :+3:.  Though i'm the author of this PR, so github won't actually let me approve it.

File: stacks/apps/exchange.yml (hunk 2 above)
Comment (kookster) [reply stacks/apps/exchange.yml:1223]: yes! which is why we still need el queue de hablar donde

File: stacks/apps/exchange.yml (hunk 1 above)
Comment (kookster) [reply stacks/apps/exchange.yml:967]: right, I don't think it has any other unique needs

**Comment by kookster** [comment]
@farski can you review this as well?
I think it makes sense, and we didn't miss anything?

**Comment by kookster** [comment]
I just merged in `main`, hopefully does something good for the build (and merging)

**Review by farski — APPROVED** [review]
Seems good

**Pull request merged**